src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const employeesCollection = document.querySelectorAll('ul > li');
+const employeesCollectionRepository = document.querySelector('ul');
```

Comment:
You could create only one variable 'employeesCollectionRepository' (strange name) and use it like `[...employeesCollectionRepository.children]` instead of creating another variable 'employeesCollection'